src/components/layout.tsx

```diff
@@ -1,5 +1,5 @@
 import Navbar from "./navbar";
-import { ReactNode } from "react";
+import { ReactNode, useState, useEffect } from "react";
```

Comment:
wird doch gar nicht benutzt

Reply:
wurde in [c2e21ff](https://github.com/top-22/forum/pull/14/commits/c2e21ff4b462bdf6eedc735deaf1474e4ecbac71) entfernt